packages/xo-web/src/xo-app/hub/resource.js

```diff
@@ -171,10 +180,66 @@ export default decorate([
       redirectToTaskPage() {
         this.props.router.push('/tasks')
       },
+      showDescription() {
+        const {
+          data: { public: _public },
+          name,
+        } = this.props
+        alert(
+          name,
+          <div>
+            {isEmpty(omit(_public, BANNED_FIELDS)) ? (
+              <p>{_('hubTemplateDescriptionNotAvailable')}</p>
+            ) : (
+              <div>
+                <Ul>
+                  {EXCLUSIVE_FIELDS.map(fieldKey => {
+                    const field = _public[fieldKey]
+                    return field !== undefined ? (
+                      <Li key={fieldKey}>{field}</Li>
+                    ) : null
+                  })}
+                </Ul>
```

Comment:
What if there isn't a `longDescription`?

Reply:
we will end up with an empty list, which has no effect on the UI